script.js

```diff
@@ -29,6 +35,11 @@ function initialise()
 	matrixDefault();
 	drawArena();
 	turnCount = 0;
+	turnCountBackup=0;
+	gameStarted = false;
+	round = 0;
+	playersEliminated = {red:false,green:false,blue:false,yellow:false};
```

Comment:
A little spacing would make code more easier to read as well as extend later on.